src/main/java/no/sikt/sws/WorkspaceIndexHandler.java

```diff
@@ -24,11 +27,19 @@ protected IndexResponse processInput(Void input, RequestInfo request, Context co
         var workspace = request.getPathParameter(WORKSPACE_IDENTIFIER);
         var action = request.getPathParameter(RESOURCE_IDENTIFIER);
 
-        var response = httpMethod + " opensearch/" +workspace + "/" + action;
+        var endpoint = OPENSEARCH_ENDPOINT_ADDRESS;
+        var region = AWS_REGION;
+
+        var response = httpMethod + " opensearch/" + workspace + "/" + action
+                + " endpoint: " + endpoint + " region: " + region;
```

Comment:
This starts looking quite messy. The original idea behind the line was to print something that would look like a valid call towards opensearch. Something like:
`POST https://search-swsopensearch-1zboikxsdfsadefrwbs9-fwehfbhwaefwe.eu-west-1.es.amazonaws.com/sondre-index1/_mapping`
Can you change the response variabel to reflect this (call it path or something)
and then log your new variabels in different println-calls?